UIKitTemplate/UIKitTemplate/Model/NotificationCellButton.swift

```diff
@@ -0,0 +1,18 @@
+// NotificationCellButton.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Модель ячейки с кнопкой
```

Comment:
```suggestion
/// Модель ячейки с кнопкой
```
сл раз за слово Модель будем отжиматься